src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int AMOUNT_INDEX = 1;
+    private static final int TYPE_INDEX = 0;
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The getStatistic method is doing too much. It's reading a file, processing the data and then writing the result to another file. According to the 'Single Responsibility Principle', a method should do one thing and do it well. Consider splitting this method into smaller ones, each with a clear, single responsibility.